### PR TITLE
Add section for comparison and logical query operators

### LINKED ISSUE #1314: Add documentation for supported query operators
### What should be done?
 Add details and examples about each supported operator. Link operators to the supported commands (references), when necessary.
Include the following sections:
* [x] Comparison query operators
* [x] Logical query operators
* [ ]  Array operators
* [ ] Bitwise query operators
* [ ] Element operators
* [ ] Evaluation operators

### PR DESCRIPTION
**Pull request by Fashander**
# Description

Closes #1314.

## Readiness checklist

* [ ] I added tests for new functionality or bugfixes.
* [ ] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: website/docs/reference/operators/comparison-operators.md [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+  }
+]
+```
+
+## $in
+
+*Syntax*: `{ field: { $in: [<value1>, <value2>, ... <valueN> ] } }`
+
+The `$in` operator selects documents that contain any of the values in a specified array.
+The following operation queries the `employees` collection for documents where the `catalog` array contains `printer` and `crayons`.
```
Comment (chilagrow) [inline website/docs/reference/operators/comparison-operators.md:287]: Should it be `printer` or `crayons`? If I recall, `$in` needs to match at least one element of the array. I think the example should show where only one of `printer` or `crayons` present. Maybe we can remove `printer` from one of the example and demonstrate the match. :hugs: If ours doesn't behave like that, maybe there is a bug.

**Review by chilagrow — COMMENTED** [review]
Thanks for this :muscle: some early comments.

File: website/docs/reference/operators/comparison-operators.md (hunk 1 above)
Comment (Fashander) [reply website/docs/reference/operators/comparison-operators.md]: PTAL

**Review by w84thesun — APPROVED** [review]

File: website/docs/reference/operators/logical-operators.md [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+   $and:[
+      {
+         price:{
+            $lt: 100
+         }
+      },
+      {
+         stock: {
+            $ne: 0
+            }
```
Comment (chilagrow) [inline website/docs/reference/operators/logical-operators.md:132]: Formatting of this `{` looks strange

**Review by chilagrow — APPROVED** [review]
Just tiny formatting comment. Looking good!

**Review by chilagrow — APPROVED** [review]

**Review by w84thesun — APPROVED** [review]

File: website/docs/reference/operators/comparison-operators.md [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+]
+```
+
+To query values in an embedded document, use [dot notation](./../../understanding_ferretdb.md#dot-notation).
+The following operation queries the `employees` collection for documents that match the field `first` in the embedded document `name`.
+
+```js
+db.employees.find({
+   "name.first":{
+      $eq:"Earl"
```
Comment (rumyantseva) [inline website/docs/reference/operators/comparison-operators.md:109]:
```suggestion
      $eq: "Earl"
```

**Review by rumyantseva — APPROVED** [review]
👍 👍 👍 

LGTM, one tiny formatting note.

**Review by AlekSi — APPROVED** [review]

**Review by noisersup — APPROVED** [review]

**Pull request merged**